app/src/page/file/view.js

```diff
@@ -30,25 +30,26 @@ class FilePage extends React.PureComponent {
   static navigationOptions = {
     title: ''
   };
-  
+
   constructor(props) {
     super(props);
     this.state = {
       mediaLoaded: false,
+      autoplayMedia: false,
```

Comment:
Should autoplay be a separate setting so the value is saved if they leave the page?

Reply:
I suppose that would be a different setting entirely. This state variable is related to the action that should be performed when the user clicks on the "Play" (download) button on the file page, instead of the usual "Download".

Reply:
Ah I gotcha.